CONTRIBUTING.md

```diff
@@ -34,6 +34,8 @@ Please add regression tests for your PR. Ideally, both positive and negative
 ones; especially so if the change is a bugfix, or touches a critical component.
 [Failure attributes can help for that](https://github.com/FStarLang/FStar/wiki/Failure-attributes).
 
+In case if build in CI fails, and you want to troubleshoot yourself, join [Slack](https://everestexpedition.slack.com) and go to [#everest-all-builds channel](https://everestexpedition.slack.com/archives/CD7QT0SES) and look for your broken build.
```

Comment:
```suggestion
In case if build in CI fails, and you want to troubleshoot yourself, join [Slack](https://aka.ms/JoinEverestSlack) and go to [#everest-all-builds channel](https://everestexpedition.slack.com/archives/CD7QT0SES) and look for your broken build.
```